Update to allow testing against Sauce Labs and Browserstack

michael-vansickle opened the pull request:
Updated Gruntfile and intern configurations to allow tests to run against saucelabs and browserstack via grunt command line settings

Commit:
- Updated Gruntfile and intern configurations to allow tests to run against saucelabs and browserstack via grunt command line settings
- Changed .travel.yml to default to testing with Saucelabs

kitsonk commented:
Can you switch over to SauceLabs before I merge this one?

kitsonk commented on typings/browser.d.ts:
```diff
@@ -0,0 +1 @@
+/// <reference path="browser/ambient/jsdom/index.d.ts" />
```
Why is this here, it seems tangental to the issue being addressed.

michael-vansickle commented:
I've updated the .travis.yml file to use Saucelabs. I also cleaned out the old jsdom stuff that made its way in when I was experimenting with something.

kitsonk commented:
Merged in ee8949753f562abab17fed29717609c16703a947
